docs/docs.html

```diff
@@ -510,8 +510,20 @@ <h5 id="config-details">Config Options</h5>
 									<code>transformHeader</code>
 								</td>
 								<td>
-									A function to apply on each header. Requires <code>header</code> to be <code>true</code>. The function receives the header as its first argument and the index as second.<br>
-									Only available starting with version 5.0.
```

Comment:
I think we should keep that this is only availalbe on the version 5.0

I also plan to do a minor version for the parameter. So the new parameter structure should be only available on 5.4+ versions. That should be noted too.

Reply:
Resolved with https://github.com/mholt/PapaParse/pull/898/commits/ac6f19168cc742ca12ef7ee3c2dcae607e9301a3